### ISSUE TITLE
Documentation for upgrading to new version

### ISSUE THREAD

**Issue by sajjadmurtaza** [refinery/refinerycms#3235]
I have following gems and their path
`gem 'refinerycms', '~> 2.0.0'`
`gem 'refinerycms-core', '~> 2.0.0', :path => 'vendor/engines'`
`gem 'refinerycms-blog', '~> 2.0.0'`
`gem 'refinerycms-akouo', :path => 'vendor/extensions'`
`gem 'refinerycms-workshops', :path => 'vendor/extensions'`
`gem 'refinerycms-slides', :path => 'vendor/extensions'`
`gem 'refinerycms-titans', :path => 'vendor/extensions'``

When i'm trying to update them facing 

> Bundler could not find compatible versions for gem "railties":
>   In Gemfile:
>     jquery-ui-rails was resolved to 5.0.5, which depends on
>       railties (>= 3.2.16)

```
jquery_mobile_rails was resolved to 1.4.5, which depends on
  railties (>= 3.1.0)
```

 so on .....

Need guide. Thanks

**Comment by bricesanchez** [comment]
`jquery_mobile_rails` looks like old, could you remove it or replace it.

Update guides are here but they will refer to 3.0: 

- http://www.refinerycms.com/guides/upgrading-to-the-latest-stable-version
- http://www.refinerycms.com/guides/upgrading-to-refinery-30

**Comment by sajjadmurtaza** [comment]
@bricesanchez my issue has been resolve by changing `jquery` version. I have following gem and their paths.
`gem 'refinerycms', '~> 2.0.0'`
`gem 'refinerycms-core', '~> 2.0.0', :path => 'vendor/engines'`
`gem 'refinerycms-blog', '~> 2.0.0'`
`gem 'refinerycms-akouo', :path => 'vendor/extensions'`
`gem 'refinerycms-workshops', :path => 'vendor/extensions'`
`gem 'refinerycms-slides', :path => 'vendor/extensions'`
`gem 'refinerycms-titans', :path => 'vendor/extensions'``
But when i'm updating refinerycms, i'm facing following issues regarding dependencies
 

> Bundler could not find compatible versions for gem "refinerycms-core":
  In Gemfile:
    refinerycms-core (~> 2.0.0)

    refinerycms (~> 3.0.0) was resolved to 3.0.0, which depends on
      refinerycms-core (= 3.0.0)

    refinerycms-blog (~> 2.0.0) was resolved to 2.0.5, which depends on
      refinerycms-core (~> 2.0.3)

    refinerycms-blog (~> 2.0.0) was resolved to 2.0.5, which depends on
      refinerycms-settings (~> 2.0.1) was resolved to 2.0.2, which depends on
        refinerycms-core (~> 2.0.1)

    refinerycms-titans was resolved to 1.0, which depends on
      refinerycms-core (~> 2.0.8)

    refinerycms-titans was resolved to 1.0, which depends on
      refinerycms-core (~> 2.0.8)

    refinerycms-titans was resolved to 1.0, which depends on
      refinerycms-core (~> 2.0.8)

    refinerycms-titans was resolved to 1.0, which depends on
      refinerycms-core (~> 2.0.8)

**Comment by bricesanchez** [comment]
@sajjadmurtaza Is it fixed for you ?
